### PR TITLE
applivery-android 0.3.0

### PR DESCRIPTION
**Pull request by CesarTrigo**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/applivery-android/0.3.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+title: Applivery.com v3 Android Deploy
+summary: Deploy your awesome Android App to Applivery.com
+description: |-
+  Deploy an Android application to [Applivery](http://www.applivery.com),
+  add notes and notify Collaborators and Employees.
+
+  Register a Applivery account at [http://www.applivery..com/](http://www.applivery.com)
```
Comment (koral--) [inline steps/applivery-android/0.3.0/step.yml]: typo `..` instead of `.`

**Comment by lszucs** [comment]
Hi @CesarTrigo 

You had an icon update in your `0.3.0` I closed earlier. Did you forget to to include it in this PR?

**Comment by CesarTrigo** [comment]
Hi @lszucs,
It was a mistake and one of the reasons why we decided to cancel both pull requests.
We still don't have the new logo totally ready. We'll do another PR just for the new logo once it's ready.
C.

**Comment by CesarTrigo** [comment]
Hi @BirmacherAkos thank you.
By the way, we just discovered that there's some kind of "verified" badge beside certain integrations (screenshot below). What would be the process to get that badge?
Captura de pantalla 2019-06-13 a las 17 27 16

**Comment by BirmacherAkos** [comment]
Hi @CesarTrigo ,

That badge means the step is developed and maintained by Bitrise.
The other "purple" one with a figure in it means it's a community step.

So as a community step developer you will not be able to get the green badge for your step.
But do not forget, we are always [hiring](https://careers.bitrise.io/) ;)

**Comment by CesarTrigo** [comment]
Thank you @BirmacherAko everything is clear now!
Best,
C.

**Pull request merged**